content/200-concepts/100-components/01-prisma-schema/04-data-model.mdx

```diff
@@ -447,12 +683,120 @@ model User {
 }
 ```
 
-> **Note**: A model can only have _one_ ID, and that ID can either be a single field or based on multiple fields.
+</tab>
+<tab>
+</tab>
+</TabbedContent>
+
+In the following example, the `User` ID is represented by a mandatory `@unique` field:
+
+<TabbedContent tabs={[<FileWithIcon text="Relational databases only"/>]}>
+<tab>
+
+```prisma highlight=2;normal
+model User {
+  email   String   @unique
+  name    String?
+  role    Role     @default(USER)
+  posts   Post[]
+  profile Profile?
+}
+```
+
+</tab>
+<tab>
+</tab>
+</TabbedContent>
+
+#### Defining IDs in MongoDB
+
+The MongoDB connector has [specific rules for defining an ID field](../../../reference/api-reference/prisma-schema-reference#mongodb) <span class="api"></span> that differs from relational databases. An ID can be defined by a single field using the [`@id`](../../../reference/api-reference/prisma-schema-reference#id) <span class="api"></span> attribute and must include `@map("_id")`.
+
+In the following example, the `User` ID is represented by the `id` string field that accepts an auto-generated `ObjectId`:
+
+<TabbedContent tabs={[<FileWithIcon text="MongoDB only" icon="prisma"/>]}>
+<tab>
+
+```prisma highlight=2;normal
+model User {
+  id      String   @id @default(dbgenerated()) @db.ObjectId @map("_id")
+  email   String   @unique
+  name    String?
+  role    Role     @default(USER)
+  posts   Post[]
+  profile Profile?
+}
+```
+
+</tab>
+<tab>
+
+
+</tab>
+
+</TabbedContent>
+
+In the following example, the `User` ID is represented by the `id` string field that accepts something other than an `ObjectId`:
+
+<TabbedContent tabs={[<FileWithIcon text="MongoDB only" icon="prisma"/>]}>
+<tab>
+
+```prisma highlight=2;normal
+model User {
+  id      String   @id @map("_id")
+  email   String   @unique
+  name    String?
+  role    Role     @default(USER)
+  posts   Post[]
+  profile Profile?
+}
+```
+
+</tab>
+<tab>
+
+
+</tab>
+
+</TabbedContent>
+
+In the following example, the `User` ID is represented by the unique `email` string field:
+
+<TabbedContent tabs={[<FileWithIcon text="MongoDB only" icon="prisma"/>]}>
+<tab>
+
+```prisma highlight=2;normal
+model User {
+  email   String   @unique @map("_id")
+  name    String?
+  role    Role     @default(USER)
+  posts   Post[]
+  profile Profile?
+}
+```
```

Comment:
`_id` does not have to be a "primary key" (defined via `@id`)?

Reply:
It does not: https://prisma-company.slack.com/archives/C013RMLGE72/p1624016268109500?thread_ts=1624005065.105200&cid=C013RMLGE72

Reply:
This changed: https://prisma-company.slack.com/archives/C013RMLGE72/p1626077485002400?thread_ts=1626077390.002300&cid=C013RMLGE72

Have removed all examples of `@unique @map("_id")`